README.md

```diff
@@ -409,6 +409,15 @@ This is the current roadmap for the CommandAPI (as of 30th April 2024):
         </tr>
     </thead>
     <tbody>
+        <tr>
+            <td valign="top"><b>9.6.0</b></td>
+            <td valign="top">???</td>
+            <td valign="top">
+                <ul>
+                    <li>https://github.com/JorelAli/CommandAPI/pull/594 Adds a config option to allow the CommandAPI to be more lenient when updating to a new minor version (e. g. from 1.21 to 1.21.1)</li>
```

Comment:
*Small nitpick*: it should be written as “(e.g. from 1.21 to 1.21.1)” instead of “(e. g. from 1.21 to 1.21.1).”

Reply:
I'd rather we use "for example" instead of "e.g." in the documentation